controller/tsconfig.json

```diff
@@ -32,5 +32,5 @@
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx"
-  ]
+, ".latestpewpewversionfile.ts", "pages/api/util/.latestpewpewversionfile.ts"  ]
```

Comment:
I don't see that second file in the tree?

Reply:
Either way it's a temporary file since as soon as the controller is replaced by a failed healthcheck or deploy we'll be back to the static one checked into the root. We really need to store the data in s3 so it will survive deploys and unhealthy instances.

Reply:
No. Most of the code stays the same. We just have to modify 2 of the functions.